### PR TITLE
Nazarovo/advanced search

### PR DESCRIPTION
**Pull request by mudabek**

### CODE REVIEW

File: app/src/main/java/edu/grinnell/appdev/grinnelldirectory/Activity/AdvancedSearchActivity.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+import android.widget.Button;
+import android.widget.Spinner;
+import android.widget.TextView;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+import butterknife.OnClick;
+import edu.grinnell.appdev.grinnelldirectory.R;
+
+
```
Comment (LarryAsante) [inline app/src/main/java/edu/grinnell/appdev/grinnelldirectory/Activity/AdvancedSearchActivity.java]: Add proper documentation to this class

File: app/src/main/java/edu/grinnell/appdev/grinnelldirectory/Activity/AdvancedSearchActivity.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    @BindView(R.id.search_button)
+    Button submitButton;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_advanced_search);
+        ButterKnife.bind(this);
+    }
+
```
Comment (LarryAsante) [inline app/src/main/java/edu/grinnell/appdev/grinnelldirectory/Activity/AdvancedSearchActivity.java:87]: If you write your own methods and make them public, you should document them

**Review by LarryAsante — CHANGES_REQUESTED** [review]
Check the comments -- some slight additions and we should be good to go

File: app/src/main/java/edu/grinnell/appdev/grinnelldirectory/Activity/AdvancedSearchActivity.java (hunk 1 above)
Comment (prabirmsp) [reply app/src/main/java/edu/grinnell/appdev/grinnelldirectory/Activity/AdvancedSearchActivity.java]: @mudabek check out @as-f 's branches - he has excellent documentation

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -1,9 +1,204 @@
 <resources>
-    <string name="app_name">Grinnell Directory</string>
-    <string name="title_activity_default_login">Sign in</string>
+<string name="app_name">Grinnell Directory</string>
+<string name="title_activity_default_login">Sign in</string>
+
```
Comment (prabirmsp) [inline app/src/main/res/values/strings.xml]: @LarryAsante is it better to put these in here, in a `Constants.java` file, or put it on a server and pull it to make it easier to change?

File: app/src/main/res/values/styles.xml [hunk 4]
```diff
@@ -1,7 +1,7 @@
 <resources>
 
     <!-- Base application theme. -->
-    <style name="AppTheme" parent="Theme.AppCompat">
+    <style name="AppTheme" parent="Base.Theme.AppCompat">
```
Comment (prabirmsp) [inline app/src/main/res/values/styles.xml]: @mudabek no need to change the theme yet...

**Review by prabirmsp — CHANGES_REQUESTED** [review]

**Comment by prabirmsp** [comment]
@mudabek make the changes and push

**Pull request closed without merging**